src/network/netdev/netdev-gperf.gperf

```diff
@@ -79,6 +80,15 @@ VXLAN.MaximumFDBEntries,     config_parse_unsigned,                0,
 VXLAN.PortRange,             config_parse_port_range,              0,                             0
 VXLAN.DestinationPort,       config_parse_destination_port,        0,                             offsetof(VxLan, dest_port)
 VXLAN.FlowLabel,             config_parse_flow_label,              0,                             0
+GENEVE.Id,                   config_parse_uint32,                  0,                             offsetof(Geneve, id)
+GENEVE.Remote,               config_parse_geneve_address,          0,                             offsetof(Geneve, remote)
+GENEVE.TOS,                  config_parse_unsigned,                0,                             offsetof(Geneve, tos)
```

Comment:
please use a proper parser function for this, and generate a parsing error log msg for invalid values